README.md

```diff
@@ -128,3 +128,11 @@ would be broken into the tuple of tokens:
  If it doesn't find a match for a rule, it backtracks to a previously marked point in the stream and tries the next available rule. If the parser runs out of rules to try, it raises a syntax error.
 
  As the parser finds matches, it adds the relevant token values to its syntax tree, which is eventually returned to the user if the input parses successfully.
+
+## Testing
+
+You can run the test suite locally using docker.
```

Comment:
Can you please document this a bit more? For example, would be helpful to mention:
* What the test suite runs aside from the built-in unit tests
* Code is automatically formatted via Black and isort in the CI build but not in local runs
* Where config for different linters lives (e.g. `pyproject.toml`, `setup.cfg`)

Reply:
Great point--I've added some additional color. Let me know what you think!

Reply:
Good! I may tweak this a bit in the next release as I'm changing the documentation overall, but this is much better, thanks.